war/package.json

```diff
@@ -39,14 +39,16 @@
     "less-loader": "11.1.0",
     "mini-css-extract-plugin": "2.7.5",
     "postcss": "8.4.21",
-    "postcss-less": "6.0.0",
     "postcss-loader": "7.2.4",
+    "postcss-scss": "^4.0.6",
     "prettier": "2.8.7",
+    "sass": "^1.62.0",
+    "sass-loader": "^13.2.2",
     "style-loader": "3.3.2",
     "stylelint": "15.4.0",
     "stylelint-checkstyle-reporter": "0.2.0",
     "stylelint-config-standard": "32.0.0",
-    "webpack": "5.78.0",
+    "webpack": "^5.79.0",
```

Comment:
Is this change needed for this PR or can this hunk be reverted and left for Dependabot to minimize the scope of this change?